### PR TITLE
Pin Minimum Deps in min-dependency tests, Hotfix  SDK-Tools for backwards compat

### PR DESCRIPTION
**Pull request by scbedd**

### CODE REVIEW

File: scripts/devops_tasks/test_regression.py [hunk 1]
```diff
@@ -23,7 +23,8 @@
     find_packages_missing_on_pypi,
     find_whl,
     find_tools_packages,
-    get_installed_packages
+    get_installed_packages,
+    extend_dev_requirements
```
Comment (praveenkuttappan) [inline scripts/devops_tasks/test_regression.py:27]: I don't see this used anywhere or am i overlooking something.

File: scripts/devops_tasks/test_regression.py [hunk 2]
```diff
@@ -38,6 +39,10 @@
 AZURE_SDK_FOR_PYTHON_GIT_URL = "https://github.com/Azure/azure-sdk-for-python.git"
 TEMP_FOLDER_NAME = ".tmp_code_path"
 
+CUSTOM_REGRESSION_INSTALLED_PACKAGES = [
```
Comment (praveenkuttappan) [inline scripts/devops_tasks/test_regression.py:42]: unused or for future reference?

File: scripts/devops_tasks/test_regression.py (hunk 1 above)
Comment (scbedd) [reply scripts/devops_tasks/test_regression.py:27]: 👍

File: scripts/devops_tasks/test_regression.py (hunk 2 above)
Comment (scbedd) [reply scripts/devops_tasks/test_regression.py:42]: I'm in the process of removing! Sec.

**Review by praveenkuttappan — APPROVED** [review]

**Review by praveenkuttappan — APPROVED** [review]

**Pull request merged**